eng/scripts/Language-Settings.ps1

```diff
@@ -90,3 +90,19 @@ function Get-AllPackageInfoFromRepo ([string] $ServiceDirectory) {
 
   return $allPackageProps
 }
+
+function Get-rust-EmitterName() {
+  return "@azure-tools/typespec-rust"
+}
+
+function Get-CrateVersion([string]$projectDirectory) {
+  return "0.1.0"
+}
+
+function Get-CrateName(){
+  return (Split-Path $projectDirectory -Leaf)
+}
+
+function Get-rust-EmitterAdditionalOptions([string]$projectDirectory) {
```

Comment:
What calls this? The contents make sense, I just don't know how it's wired into the build.

Reply:
/cc @hallipr